### PR TITLE
AAP-19263 create inventory and copy to all controllers

### PR DESCRIPTION
**Pull request by qjones7**
This PR implements the configuration of the inventory file and copies the file to all controller instances.

JIRA issue for reference: [https://issues.redhat.com/browse/AAP-19263](https://issues.redhat.com/browse/AAP-19263)

Steps to test:

Pull down the pr

Deploy resources using steps mentioned here [(https://github.com/ansible-content-lab/aws_terraform_deployment#deploying-infrastructure](https://github.com/ansible-content-lab/aws_terraform_deployment#deploying-infrastructure)

ssh into controller and check for inventory file

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -147,6 +154,27 @@ variable "infrastructure_public_key_filepath" {
...
+
+variable "aap_admin_password" {
+  description = "The admin password to create for Ansible Automation Platform application."
+  type = string
+  sensitive = true
+}
+
+variable "infrastructure_aap_installer_inventory_path" {
+  description = "Inventory path on the installer host"
+  default = "/home/ec2-user/inventory_aws.txt"
```
Comment (leena-jawale) [inline variables.tf]: can we remove .txt from here? Can we just haev `/home/ec2-user/inventory_aws`

File: main.tf [hunk 2]
```diff
@@ -178,4 +178,33 @@ resource "aws_key_pair" "admin" {
...
+        aap_red_hat_password= var.aap_red_hat_password
+        aap_db_host = module.database.infrastructure_controller_rds_hostname
+        aap_admin_password = var.aap_admin_password
+        infrastructure_admin_username = var.infrastructure_admin_username
+      })
+      destination = var.infrastructure_aap_installer_inventory_path
+  }
+    
+
+
```
Comment (leena-jawale) [inline main.tf]: remove extra lines

File: variables.tf (hunk 1 above)
Comment (qjones7) [reply variables.tf]: complete

File: main.tf (hunk 2 above)
Comment (qjones7) [reply main.tf]: complete

**Review by leena-jawale — APPROVED** [review]

**Pull request merged**